packages/arb-avm-cpp/cavm/cexecutioncursor.h

```diff
@@ -18,7 +18,7 @@
 
 #include "ctypes.h"
 
-#include <stdint.h>
```

Comment:
These should work as c headers for the cgo build which doesn't include `cstdint`

Reply:
whoops, I knew that :)